contracts/HubPool.sol

```diff
@@ -405,6 +433,14 @@ contract HubPool is HubPoolInterface, Testable, Lockable, MultiCaller, Ownable {
         delete refundRequest;
     }
 
+    function claimProtocolFeesCaptured(address l1Token) public nonReentrant {
+        if (unclaimedAccumulatedProtocolFees[l1Token] > 0) {
```

Comment:
do we need this check..? whats the harm of sending 0, the caller will jsut pay gas

Reply:
it's cheaper. it's just for gas optimisation.

I agree with you though that in this context we can remove this. In the context of the fee allocation I think it's better to keep this to remove gas usage.